### ISSUE TITLE
[469] sometimes "SPREAD" reports incorrectly.

### ISSUE THREAD

**Issue by no0nehere** [OldUnreal/UnrealTournamentPatches#208]
Just wondering if anyone else has seen a problem with UT's inbuilt SPREAD Function for DM Games.
on several occasions it has incorrectly indicated the number of Frags a player is leading by.
how I assume it should work is like this.

player1 = 10 frags , spread +5
player2 = 5   frags , spread -5
player3 = 3   frags , spread -7

**Comment by Chamberly** [comment]
Is this just from the SmartDM/SmartTDM scoreboard?

**Comment by TankBeef** [comment]
> Is this just from the SmartDM/SmartTDM scoreboard?

This is exactly why I stopped using the scoreboards, even the latest SmartSB ones. The only reason, otherwise they are great. Why is this happening?

**Comment by SeriousBuggie** [comment]
You need be more specific.
What wrong?
How this look like?
What you expected?
What you got?
How this reproduce?

**Comment by TankBeef** [comment]
> You need be more specific.
> What wrong?
> How this look like?
> What you expected?
> What you got?
> How this reproduce?

Well, to be honest, I still don't know how to properly reproduce it, because it is very random. I do know it only happens to me with the smart scoreboards, never without them. That is my personal experience, I don't know if no0nehere has the same, or if he is also using the scoreboards. The thing is, I play deathmatch mostly, you can play 4 or 5 matches with no flaws, then at let's say the 6th match aprox. you watch the spread +3 for example, and you think you are on top, but when you bring up the scoreboard, not only are you behind, but the spread +3 stays frozen. I may take a screenshot and post it the next time it shows up.

**Comment by stijn-volckaert** [comment]
This probably happens because the player score and the player frag count are not necessarily the same (this can happen due to suicides, for example) and because the score and frag count update independently of each other. The reason why they don't always update at the same time is that the base engine only keeps track of the player score, whereas SmartCTF/SmartDM tracks the player frag count. The spread is calculated based on player scores (and not based on frags). 

In other words, this is not really a bug. If you want spreads based on frag counts, SmartCTF/DM will need an update.
